.github/workflows/release.yml

```
@@ -13,7 +13,7 @@ env:
   rust_version: 1.70.0
 
 name: Release smithy-rs
-run-name: ${{ github.workflow }} ${{ inputs.semantic_version }} (${{ inputs.commit_sha }}) - ${{ inputs.dry_run && 'Dry run' || 'Production run' }}
+run-name: ${{ github.workflow }} ${{ inputs.stable_semantic_version }} ${{ inputs.unstable_semantic_version }} (${{ inputs.commit_sha }}) - ${{ inputs.dry_run && 'Dry run' || 'Production run' }}
```

```suggestion
run-name: ${{ inputs.dry_run && 'Dry run' || 'Prod run' }} - ${{ github.workflow }} ${{ inputs.stable_semantic_version }}/${{ inputs.unstable_semantic_version }} (${{ inputs.commit_sha }})
```
The dry run / prod run part always gets cut off in the GitHub UI, and is kind of important, so I suggest rearranging this. Doesn't matter if the commit hash is cutoff since only the first bit of it is needed.

---

Great suggestion! While the dry-run mode is a default, it's helpful for users to know in which mode a workflow will be executed without getting cut off in the UI.